BinaryObjectScanner/Protection/InterLok.cs

```diff
@@ -0,0 +1,30 @@
+using BinaryObjectScanner.Interfaces;
+using SabreTools.Serialization.Wrappers;
+
+namespace BinaryObjectScanner.Protection
+{
+    public class InterLok : IExecutableCheck<PortableExecutable>
+    {
+        /// <inheritdoc/>
+        public string? CheckExecutable(string file, PortableExecutable pex, bool includeDebug)
+        {
+            // Get the .rsrc section strings, if they exist
+            if (pex.GetFirstSectionStrings(".rsrc") != null)
+            {
+                // Found in "nfsc_link.exe" in IA item "nfscorigin".
+                // Full string: 
+                // (: ) InterLok PC v2.0, PACE Anti-Piracy, Copyright (C) 1998, ALL RIGHTS RESERVED
+                var match = pex.GetFirstSectionStrings(".rsrc").Find(s => s.Contains("InterLok") && s.Contains("PACE Anti-Piracy"));
```

Comment:
Since you're getting the strings in 2 places in the same method, please assign the first instance of `GetFirstSectionStrings` to a variable. This is why you're seeing the "dereference" warnings below.

Reply:
Should be fixed now.